.github/workflows/cd.yml

```diff
@@ -34,40 +34,87 @@ jobs:
             artifact_prefix: linux
             audio_backend: alsa
             target: x86_64-unknown-linux-gnu
+          - build_target: linux-armhf
+            os: ubuntu-18.04
```

Comment:
Since "bionic" is needed when adding the Ubuntu Ports source later, this must be hard-coded to `ubuntu-18.04`.